src/dvclive/live.py

```diff
@@ -87,12 +86,13 @@ def __init__(
         self._exp_name: Optional[str] = None
         self._experiment_rev: Optional[str] = None
         self._inside_dvc_exp: bool = False
-        self._dvc_repo = None
+        self._dvc_repo = get_dvc_repo()
```

Comment:
Sorry, just different code styles. I tend to lean towards a more functional code style where I try to avoid stateful changes within other class methods since it's less clear to me without searching the code when and how `self._dvc_repo` gets set.

Anyway, I changed it back here and below.